packages/e2e/local-network/scripts/make-babbage.sh

```diff
@@ -178,10 +178,11 @@ jq --raw-output ".protocolConsts.protocolMagic = ${NETWORK_MAGIC}" "${ROOT}/gene
 
 rm "${ROOT}/genesis/byron/genesis-wrong.json"
 
-jq -M '
+jq -M "
 . + {
   activeSlotsCoeff: 0.1,
   epochLength: 1000,
+  maxLovelaceSupply: $((MAX_SUPPLY * 3)),
```

Comment:
This variable is clearly not the max supply :sweat_smile:
```suggestion
  maxLovelaceSupply: $((??? * 3)),
```

Reply:
That's what I think every time I read that variable name 😄 

I'll merge the PR as is (otherwise it will increase the merge conflicts rather than reducing them).
Once this will be merged in `conway-era` as well the two branches will be aligned and we can change this variable name safely.
Next time I'll put my hands on these files, I'll fix this as well.
In the mean while, I'll seek for the right name for it.